implementations/rust/ockam/ockam_api/src/authenticator/enrollment_tokens/authenticator.rs

```diff
@@ -12,6 +12,7 @@ pub(super) const MAX_TOKEN_DURATION: Duration = Duration::from_secs(600);
 #[derive(Clone)]
 pub struct EnrollmentTokenAuthenticator {
     pub(super) trust_context: String,
+    // TODO: Replace with something sane and standard + implement expiration
```

Comment:
@nazmulidris that looks that a good issue to create

Reply:
Do you think we should first move to SQLite? Or maybe not?